isseven/checkers.py

```diff
@@ -34,6 +34,8 @@ def is_the_word_seven(possible_seven: str) -> IsSevenResult:
     tidied_possible_seven = possible_seven.strip().lower()
     if tidied_possible_seven in sevens:
         return yep(sevens[tidied_possible_seven])
+    if tidied_possible_seven[::-1] in sevens:
+        return yep(sevens[tidied_possible_seven] + " backwards")
```

Comment:
Now that I think about it does this work? I think you'd need to use the reversed string as the dictionary key

Reply:
omg you're so spoiled with code that works

Reply:
my goal for the bar of quality is always: "marcus + 1"

Reply:
WTYD